data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+# Copyright (c) 2020 by Fernando S. Sáez
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "The Box 1"
+	landing
+	name "The Sheragi bunker"
+	description "Visit Albert Foster on Zug to see what he discovered."
+	source
+		near "Zubeneschamali" 1 100
+	destination Zug
+	
+	to offer
+		has "event: rim archaeology results"
+		has "First Contact: Unfettered: declined"
```

Comment:
```suggestion
```
Is this event even possible? I couldn't find it in any of the game files and to proceed with the campaign, I had to delete this line from my local game files.

Reply:
This comes from the short mission that happens when you land on an Unfettered planet. There is some conversation, and some insight on them, and at the end of it, since it doesn't continue, the mission is declined. I placed this there as a requirement because that is the part that mentions the Sheragi, and makes it obvious that the Hai knows about them.